README.md

```diff
@@ -59,3 +66,15 @@ $ python -m myproject
 
 Hello world!
 ```
+
+## Release flow
+
+To create new release:
+
+1. Merge all changes to the `main` branch
+1. Navigate to Repo => Actions
```

Comment:
Increment numbers

Reply:
No need. MD handles it correctly
![Uploading Screenshot 2022-06-29 at 10.13.15.png…]()